### ISSUE TITLE
provisioning fails for STM TPM

### ISSUE THREAD

**Issue by rvka** [tpm2-software/tpm2-tss#1952]
On my Lenovo ThinkPad X1C6 with STM TPM, tss2_provision fails with

```
ERROR:fapi:src/tss2-fapi/fapi_crypto.c:2036:ifapi_verify_ek_cert() unable to get issuer certificate
ERROR:fapi:src/tss2-fapi/fapi_crypto.c:2037:ifapi_verify_ek_cert() ErrorCode (0x00060001) Failed to verify EK certificate
ERROR:fapi:src/tss2-fapi/api/Fapi_Provision.c:823:Fapi_Provision_Finish() ErrorCode (0x00060001) Verify EK certificate
ERROR:esys:src/tss2-esys/esys_iutil.c:389:iesys_handle_to_tpm_handle() Error: Esys invalid ESAPI handle (ff).
ERROR:esys:src/tss2-esys/esys_iutil.c:1104:esys_GetResourceObject() Unknown ESYS handle. ErrorCode (0x0007000b)
ERROR:esys:src/tss2-esys/api/Esys_FlushContext.c:138:Esys_FlushContext_Async() flushHandle unknown. ErrorCode (0x0007000b)
ERROR:esys:src/tss2-esys/api/Esys_FlushContext.c:66:Esys_FlushContext() Error in async function ErrorCode (0x0007000b)
ERROR:fapi:src/tss2-fapi/fapi_util.c:1079:ifapi_session_clean() Cleanup session failed.
ERROR:fapi:src/tss2-fapi/api/Fapi_Provision.c:168:Fapi_Provision() ErrorCode (0x00060001) Provision
Fapi_Provision(0x60001) - fapi:Catch all for all errors not otherwise specified
```

This appears to be due to STM root certificates in `tss2-fapi/fapi_certificates.h` not being self-signed.  As a result, the certificate chain doesn't terminate in a self-signed trusted certificate, and OpenSSL's `X509_verify_cert` refuses to verify it.

**Comment by rvka** [comment]
Submitted #1953

**Comment by AndreasFuchsSIT** [comment]
Fixed via #1953 and merged.
